backend/eventloop/shared_resource_loop/sharedresourceloop_managedenv_test.go

```diff
@@ -512,7 +512,15 @@ var _ = Describe("SharedResourceEventLoop Test", func() {
 
 		It("should clean up Application database entries on managed environment deletion, and create operations for those", func() {
 
-			_, _, _, instance, _, err := db.CreateSampleData(dbQueries)
+			_, _, engineCluster, _, _, err := db.CreateSampleData(dbQueries)
+			Expect(err).To(BeNil())
+			instance := &db.GitopsEngineInstance{
+				Gitopsengineinstance_id: "test-fake-instance-id",
+				Namespace_name:          "gitops-service-argocd",
+				Namespace_uid:           "test-fake-instance-namespace-914",
+				EngineCluster_id:        engineCluster.Gitopsenginecluster_id,
+			}
+			err = dbQueries.CreateGitopsEngineInstance(ctx, instance)
```

Comment:
Possible to use the GitOpsEngineInstance coming from `CreateSampleData`?

Reply:
same reason as above